change.js

```diff
@@ -1,47 +1,26 @@
 //  When you click on one of the car elements, change the width of the border to a higher value, and change the background color to any other color of your choosing.
-var cards = ['carCard0', 'carCard1', 'carCard2'];
-var change = document.getElementById('cards[0]');
-// var clickMe = document.getElementById('clickMe');
- //Keeping it unobstrusive
-document.getElementById("carCard0").addEventListener("click", transform);
-// document.getElementById("carCard1").addEventListener("click", transform1);
-// document.getElementById("carCard2").addEventListener("click", transform2);
-
-function transform() {
-
-  carCard0.style.backgroundColor = '#ddd';
-  carCard0.style.width = '300px';
-  carCard0.style.borderColor = 'orange';
-  carCard0.style.color = 'orange';
-  document.getElementById('edit').focus();
-  var input = document.getElementById('edit');
-   if (input.value.length !== 0)
-       input.value = '';
+var cards = document.getElementsByClassName('carCard');
```

Comment:
The following code should be wrapped in the activateEvents function. The reason you aren't able to click on cards is that the listeners are being applied before the page loads.